### DISCUSSION TITLE
how is "pose" (skeleton) added?

### DISCUSSION THREAD

**Discussion by movelikeriver** [Acly/krita-ai-diffusion#1079] [Q&A] [answered]
following the instructions from https://www.youtube.com/watch?v=-QDPEcVmdLI
where can i find the "Pose" (skeleton or motion) in the Krita menu?  any extra plugin?
<img width="932" alt="image" src="https://github.com/user-attachments/assets/f4055fa4-deaa-40c0-bc1f-d22dfe887b68">

same in https://www.youtube.com/watch?v=AF2VyqSApjA
<img width="912" alt="image" src="https://github.com/user-attachments/assets/fb1f0fb6-6d70-4588-8acc-e09f928ff1cd">

**Comment by ETCore7** [comment, marked as answer]
It's a controlnet "layer" that you add. 

Say you have a figure in live mode. You add a controlnet and change it to type 'pose'.
![image](https://github.com/user-attachments/assets/a1690696-4f28-4a0f-ae37-6006b28baeea)
![image](https://github.com/user-attachments/assets/5fd6412e-3ded-4952-b49c-791aa175b5d6)

Then you click the tiny magic button on the right, a vector layer is made with a bone figure.
![image](https://github.com/user-attachments/assets/dba796f8-894a-4749-9725-d6f227ffbf3a)
![image](https://github.com/user-attachments/assets/4894ea4f-88b1-4e9a-9797-d3dedcb1306c)

By moving nodes the figure should update to the new pose, but with default settings the weights are too strong. You'll have to play with settings. 

While trying with the new version the preview image seems to include the bone figure visually instead of ignoring it. It's not very intuitive.
![image](https://github.com/user-attachments/assets/bdd3a499-a768-4576-baf0-740a571c1278)

**Reply by movelikeriver** [reply]
thank you for the detailed answer!  after installing controlnet pose, the bone figure can be generated now

**Reply by geiseri** [reply]
If I have an openpose json file how would I import that?
